js/blocks/loader/edit.js

```diff
@@ -1,67 +0,0 @@
-import inspectorControls from './inspector'
-import inspectorAdvancedControls from './advanced'
-import controls from '../controls';
-import { simplifiedFields } from "./fields";
-import icons from '../../../assets/icons.json';
-
-const { applyFilters } = wp.hooks;
-const { ServerSideRender } = wp.editor;
-
-const formControls = ( props, block ) => {
-
-	const fields = simplifiedFields( block.fields ).map( (field, index) => {
-
-		// If its not meant for the editor then continue (return null).
-		if ( field.location && !field.location.includes( 'editor' ) ) {
-			return null
-		}
-
-		const loadedControls = applyFilters( 'block_lab_controls', controls );
-		const controlFunction = field.controlFunction || loadedControls[ field.control ];
-		const control = typeof controlFunction !== 'undefined' ? controlFunction( props, field, block ) : null;
-
-		return (
-			<div key={field.name + "-" + index}>
-				{control}
-			</div>
-		)
-	} )
-
-	return (
-		<div key={ block.name + "-fields" }>
-			{fields}
-		</div>
-	)
-}
-
-const editComponent = ( props, block ) => {
```

Comment:
This file is mainly refactored. For example, `editComponent` is moved to `components/edit.js`